transformer_engine/pytorch/softmax.py

```diff
@@ -222,6 +212,4 @@ def forward_torch_softmax(
     @staticmethod
     def get_batch_per_block(sq: int, sk: int, b: int, np: int) -> int:
```

Comment:
Why do I as a user need to think about this thing? That should be hidden from me!

Reply:
This is in `softmax.py`, which is not directly user facing. It is required to figure out if the fused softmax kernel for the given configuration (batch size, sequence length etc.) is available or if the framework version should be used.